forge-gui/res/cardsfolder/upcoming/scavenged_brawler.txt

```diff
@@ -0,0 +1,16 @@
+Name:Scavenged Brawler
+ManaCost:6
+Types:Artifact Creature Construct
+PT:4/4
+K:Flying
+K:Vigilance
+K:Trample
+K:Lifelink
+A:AB$ Pump | Cost$ 4 ExileFromGrave<1/CARDNAME> | ActivationZone$ Graveyard | ValidTgts$ Creature | SubAbility$ DBPutCounter | SorcerySpeed$ True | CounterType$ P1P1 | SpellDescription$ Choose target creature. Put four +1/+1 counters, a flying counter, a vigilance counter, a trample counter, and a lifelink counter on that creature. Activate only as a sorcery.
+SVar:DBPutCounter:DB$ PutCounter | CounterType$ P1P1 | CounterNum$ 4 | Defined$ Targeted | SubAbility$ DBFlyingCounter
```

Comment:
They should all be put together

Reply:
hmm that's only possible if you have to choose between counters right. Can't find a similar card where all are together

Reply:
oh but I guess the problem is that this will trigger cards that care when counters are put upon something multiple times

Reply:
_Unexpected Fangs_ is one

Reply:
yea that's the reason, you can't cast this to draw 5 cards with _Generous Patron_ in play :P

Reply:
hmm now it gives every counter type 4 times
![image](https://user-images.githubusercontent.com/67333662/198833194-8d506370-a8be-4b50-b264-e5e9029e3b05.png)

Reply:
You probably can't use CounterNum but CounterTypes should be a List so just repeating some would be ok for now

Reply:
ah I see what you mean, thanks !